lib/apex-class-cleanup.js

```diff
@@ -5,6 +5,7 @@ const handlebars = require('handlebars');
 const promptly = require('promptly');
 
 const ApexClassMetadataType = 'ApexClass';
+const FallbackPollTimeout = 10 * 10000;
```

Comment:
if the default is 100 seconds i think that's more clearly expressed as 100 * 1000 right?
maybe also add a comment after the multiplication to say for clarity
// 100 seconds

Reply:
@bruce-lindsay http://jsforce.github.io/jsforce/doc/api_metadata.js.html - it looks the like the timeout is measured in milliseconds, and defaults to just a single second.  Wouldn't it take 100,000 milliseconds to equal 100 seconds?

Reply:
i'm just sayin, the unit conversion is 1000 ms per 1 second. The value is correct, but you'd want to keep the unit conversion as a literal value in the code so people recognize it vs the value of the other unit.  that's why i was proposing the 100 show as the first literal for number of seconds, and the 1000 show as the next literal for number of ms per second. Now it says 10 * 10000 which, neither of those is a number that is going to be obvious